packages/reframed/src/reframed.ts

```diff
@@ -502,7 +502,7 @@ function monkeyPatchIFrameEnvironment(iframe: HTMLIFrameElement, shadowRoot: Ref
 	// Event listeners for these events should be added normally
 	// instead of being redirected to other event targets.
 	// TODO: there are probably a lot more events we don't want to redirect, e.g. "pagehide" / "pageshow"
-	const nonRedirectedEvents = ['DOMContentLoaded', 'popstate', 'unload'];
+	const nonRedirectedEvents = ['DOMContentLoaded', 'popstate', 'unload', 'load'];
```

Comment:
adding `load` here isn't actually necessary since we're attaching the listener to the iframe element (which still uses the original `addEventListener()`) and not its window (whose `addEventListener()` we've replaced with a proxy)